spec/helpers/posts_helper_spec.rb

```diff
@@ -17,6 +17,13 @@
         post_page_title(post)
       end
     end
+
+    context "with a reshare" do
+      it "returns 'Reshare by...'" do
```

Comment:
A side note (off topic here, because it has nothing to do with the feed): I don't know why you added this test here, because it is never used in the feed, the feed contains always the original root post with the original title. But it tests the current behavior of `post_page_title` and more tests are never bad, so it's OK ;)

I don't know if we should change the `post_page_title` for reshares, because for notifications  it's sometimes suboptimal. 
* Likes for example look like this:
  "[[person x]]() has liked your post [Reshare by [myname]]()."
  (I know that it's my reshare, but would be useful to know which one)
* For comments on reshares of another person it looks like this:
  "[[person x]]() also commented on [person y]’s post [Reshare by [person y]]()."
  (It's a bit redundant, "... on [person y]’s reshare of [person y]'s post" would be better.)

And for the single post view of a reshare the "Reshare by ..." is only displayed shortly, but some javascript replaces it immediately with the title of the original title of the post.

Should we create a refactoring issue for `post_page_title`?